src/middlewared/debian/control

```diff
@@ -21,6 +21,7 @@ Build-Depends: alembic,
                python3-croniter,
                python3-cryptit,
                python3-dbus,
+               python3-digitalocean,
```

Comment:
Why is this needed?

Reply:
good catch, it's not necessary to specify explicitly as it's already pulled as a dependency of python3-certbot-dns-digitalocean.